client/src/utils/styles/Home.css

```diff
@@ -0,0 +1,8 @@
+.background {
```

Comment:
Remove this file if you are not using it.

Reply:
ok I will